points/activities/data_selecta.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pandas as pd
+
+#
+# This program requires to launch data from https://archive-beta.ics.uci.edu/ml/datasets/daily+and+sports+activities
+# Unzip the data and copy data_selecta.py in a same directory data folder
+# Launch 'python data_selecta.py'
+#
+
+# Will be auto-saved as a xz compressed pickle file
+result_file = 'activities.xz'
```

Comment:
I don't know if there is a convention, whether the name should be activities.xz, activities.pkl (with an explicit compression parameter), or activities.pkl.xz. A quick search found all three, so I guess your name is fine.

Reply:
I don't know... I explained it in the README.md, but maybe `.pkl.xz` is more explicit. Tell me your preference